src/style.css

```diff
@@ -0,0 +1,22 @@
+
+.block1 {
+  width: 100%;
+  height: 100px;
+  background: red;
+}
+
+.block2 {
```

Comment:
it is not a good idea to identify elements using numbers, give the name that describes the content or behavior of the element